recipe/conda_build_config.yaml

```diff
@@ -2,3 +2,6 @@ c_compiler_version:    # [linux]
 - '11'                 # [linux]
 cxx_compiler_version:  # [linux]
 - '11'                 # [linux]
+
+libabseil:             # [unix]
+- '20211102.0'         # [unix] 
```

Comment:
You need to add the same for libabseil_static

Reply:
No, it didn't get picked up because you added pins in meta.yaml. The rerender will only pick up completely unpinned lines (well, except selectors) that have an entry in the global pinning or an ongoing migration.